BREAKING.md

```diff
@@ -952,18 +951,14 @@ As part of the Fluid Data Library (FDL) and Fluid Component Library (FCL) split
 -   `IComponentSerializer `will become `IFluidSerializer`
 -   `IComponentTokenProvider` will become `IFluidTokenProvider`
 
-`IComponent` will also become `IFluidObject`, and the mime type for for requests will change from `fluid/component` to `fluid/object`
```

Comment:
please keep breaking.md changes the way they were. It's a historic document showing progression over time.
Instead, you can add new entry under 0.41 saying that fluid/component is no longer supported.

Reply:
Sure, Done!